bm.js

```diff
@@ -7,7 +7,12 @@ var fs = require('fs');
 var argv = require('minimist')(process.argv.slice(2));
 
 var userhome = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
-var rc_path = userhome+"/.bmndrrc";
+if (process.env.XDG_CONFIG_HOME) {
+  var rc_path = process.env.XDG_CONFIG_HOME+"/.bmndrrc";
```

Comment:
Sounds like a good approach. How would you implement that?

Reply:
Upon second thought ...

While it makes sense to start the folder off with a period when it is directly in the home directory - so as to have it default hidden, doing so while under `XDG_CONFIG_HOME` (which should point to `$HOME/.config` unless configured otherwise) for the same reason makes less sense.

`$HOME/.config` is the hidden folder and it subfolders need not be to avoid cluttering a listing of `$HOME`.

Reply:
So change 
` var rc_path = process.env.XDG_CONFIG_HOME+"/.bmndrrc";`
to 
` var rc_path = process.env.XDG_CONFIG_HOME+"/bmndrrc";`
?